akka-persistence/src/main/mima-filters/2.5.22.backwards.excludes

```diff
@@ -0,0 +1,2 @@
+# #26451 Consolidate duplicate persistence sharding function
```

Comment:
The file name should be 2.5.21.backwards.excludes
Latest released version

Reply:
I meant to confirm that and forgot! Thanks, fixing.